### ISSUE TITLE
Feature suggestion: Audio Receptor.

### ISSUE THREAD

**Issue by IncognitoAnkh** [Zaggy1024/LC_OpenBodyCams#21]
A way to receive the monitored player audio, by making bodycams have an integrated microphone or transforming the walkie talkies somehow into microphones.

**Gameplay Limitations**
It can be limited to players with a walkie talkie enabled or inclusive only if the signal translator is unlocked.

**Why?**
This could greatly improve the operator experience, as many fun moments are lost when you are the player on the ship and you only can hear your team mates voices when they press the walkie talkie button.

**Comment by Tomveloper** [comment]
i agree

**Comment by SilverSoggy** [comment]
Yeah, that would be a really nice addition. Is it really that hard to implement?
